lib/roo_on_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module RooOnRails
-  VERSION = '1.13.1'.freeze
+  VERSION = '1.14.0'.freeze
```

Comment:
Best to do the version updates in a releasing PR

Reply:
Makes sense. Do you think for now I can merge like it is or should I split it?

Reply:
I'd split it, but I'm not well versed in releasing.

(You'll also wanna do a changelog entry)